pkg/network/render.go

```diff
@@ -833,6 +856,110 @@ func renderNetworkPublic(manifestDir string) ([]*uns.Unstructured, error) {
 	return manifests, nil
 }
 
+// renderNetworkingConsolePlugin renders the common objects related to the networking console plugin resources
+func renderNetworkingConsolePlugin(manifestDir string, cl cnoclient.Client) ([]*uns.Unstructured, error) {
+	if !isConsolePluginCRDExist(cl) {
+		log.Printf("consoleplugins.console.openshift.io CRD does not exist yet")
+		return nil, nil
+	}
+
+	tmpFallbackConsoleImage := "quay.io/orenc/networking-console-plugin:4.16"
```

Comment:
looks like in an hypershift hosted cluster, the CNO deployment on the hosted control plane is not getting populated with the `NETWORKING_CONSOLE_PLUGIN_IMAGE` env var.
could you please advise how can we inject it in that case?